pyscf/adc/radc.py

```diff
@@ -28,6 +28,8 @@
 from pyscf.adc import dfadc
 from pyscf import __config__
 from pyscf import df
+from pyscf import symm
+import pandas as pd 
```

Comment:
I don't see pandas being used and IMO it shouldn't be here in the first place. Please check and remove.

Reply:
Done